### DISCUSSION TITLE
Upon unlocking, the focus immediately shifts to the search panel.

### DISCUSSION THREAD

**Discussion by ForestQ** [keepassxreboot/keepassxc#10462] [Ideas]
I used to find it more convenient when, upon unlocking the database, the cursor was already in the search panel, allowing for immediate searching. Now, I have to manually click into the search panel with the mouse. Where can I adjust this behavior?

**Comment by michaelk83** [comment]
#10405

**Comment by JohnMilazzo** [comment]
Very much missing this feature.  Please bring it back!  🤞
